### PR TITLE
Add more .github files

### PR DESCRIPTION
**Pull request by IRONM00N**
added `CODE_OF_CONDUCT.md`, `SECURITY.md`, and `SUPPORT.md` 
moved `CONTRIBUTING.md` to .github
updated `README.md` 
added some of my files to `CODEOWNERS`

### CODE REVIEW

**Comment by IRONM00N** [comment]
> why do we put that in a another file that noone will read? Wouldnt it be better in the readme.md?

I changed some things, what is your opinion now?

**Comment by jani270** [comment]
> > why do we put that in a another file that noone will read? Wouldnt it be better in the readme.md?
> 
> I changed some things, what is your opinion now?

Its better now, but i still dont see a point behind that. 
Another question why did you change the release thing to neu repo?

**Comment by IRONM00N** [comment]
> > > why do we put that in a another file that noone will read? Wouldnt it be better in the readme.md?
> > 
> > 
> > I changed some things, what is your opinion now?
> 
> Its better now, but i still dont see a point behind that. Another question why did you change the release thing to neu repo?

because we don't have control over the other one

**Comment by IRONM00N** [comment]
> > > > > why do we put that in a another file that noone will read? Wouldnt it be better in the readme.md?
> > > > 
> > > > 
> > > > I changed some things, what is your opinion now?
> > > 
> > > 
> > > Its better now, but i still dont see a point behind that. Another question why did you change the release thing to neu repo?
> > 
> > 
> > because we don't have control over the other one
> 
> We dont need control over it? We can just ask

Moulberry has 0 interest in NEU anymore, we should not need to ask him to create github releases for us every time when we can just as easily do it ourselves.

**Comment by jani270** [comment]
> > > > > > why do we put that in a another file that noone will read? Wouldnt it be better in the readme.md?
> > > > > 
> > > > > 
> > > > > I changed some things, what is your opinion now?
> > > > 
> > > > 
> > > > Its better now, but i still dont see a point behind that. Another question why did you change the release thing to neu repo?
> > > 
> > > 
> > > because we don't have control over the other one
> > 
> > 
> > We dont need control over it? We can just ask
> 
> Moulberry has 0 interest in NEU anymore, we should not need to ask him to create github releases for us every time when we can just as easily do it ourselves.

I know that he has 0 interest in it anymore, but in times of ratters, moul is the only trusted person by the community.

**Comment by IRONM00N** [comment]
let's continue this conversation on discord, I am getting irritated by the email notifications.

**Review by romangraef — CHANGES_REQUESTED** [review]
Read from [Discord Message](https://canary.discord.com/channels/516977525906341928/755816367827058749/984935612345884692) to [Discord Message](https://canary.discord.com/channels/516977525906341928/755816367827058749/984937426327838752)

**Review by romangraef — APPROVED** [review]

**Review by romangraef — APPROVED** [review]

**Review by NopoTheGamer — DISMISSED** [review]
ok? i think

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /src/main/java/io/github/moulberry/notenoughupdates/recipes/* @romangraef
 /src/main/java/io/github/moulberry/notenoughupdates/miscgui/GuiItemRecipe.java @romangraef
 /*gradle* @romangraef
+/README.md @IRONM00N
+/.github/CODE_OF_CONDUCT.md @IRONM00N
+/.github/CONTRIBUTING.md @IRONM00N
```
Comment (jani270) [inline .github/CODEOWNERS:6]: wrong path 😃

**Review by jani270 — CHANGES_REQUESTED** [review]

**Comment by jani270** [comment]
also isnt the CONTRIBUTING.md technically from nea? You just made a change to it?

File: README.md [hunk 2]
```diff
@@ -27,18 +23,16 @@
         <img src="https://img.shields.io/discord/516977525906341928?label=discord&color=informational&logo=Discord&logoColor=FFFFFF" alt="discord">
     </a>
     <!-- downloads -->
-    <a href="https://github.com/Moulberry/NotEnoughUpdates/releases" target="_blank">
```
Comment (jani270) [inline README.md:30]: Do you really want to keep this?

**Review by jani270 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (IRONM00N) [reply README.md:30]: yes, I think we should update it to neu/neu once we make a release

Comment (jani270) [reply README.md:30]: why dont we do it now

Comment (IRONM00N) [reply README.md:30]: ig, I just don't want to be sending notifications to people that are following the repo saying that there is a new release when there really isn't

Comment (jani270) [reply README.md:30]: but we dont do releases here either?

**Review by jani270 — APPROVED** [review]
👍

**Comment by IRONM00N** [comment]
imma admin abuse

**Pull request merged**